pages/docs/installation/installation-source-dependencies.md

```diff
@@ -257,6 +258,16 @@ Keep in mind that already [PETSc](installation-source-dependencies.html#petsc) s
 Make sure that PETSc, preCICE, and your solvers are all compiled with the same MPI version!
 {% endimportant %}
 
+### Ginkgo
+
+| preCICE             | 3.2.0 >= |
+| Required Ginkgo     | 1.8.0 >= |
+| Required Kokkos     | 4.1.0 >= |
+
+[Ginkgo](https://ginkgo-project.github.io/) enables support for GPU- and OpenMP-accelerated (global) radial-basis function mappings. To enable the feature in preCICE, use the CMake option `-DPRECICE_FEATURE_GINKGO_MAPPING=ON`. Using this feature in preCICE requires additionally [Kokkos](https://kokkos.org/).
+
+Both packages follow the usual CMake build instructions and can be downloaded on the GitHub release page for [Ginkgo](https://github.com/ginkgo-project/ginkgo/releases) and [Kokkos](https://github.com/kokkos/kokkos/releases). For Kokkos, system packages might be available through your package manager as well.
+
```

Comment:
Maybe a useful hint would be how to deactivate building tests and examples for Ginkgo, which takes a while.

Reply:
yes